assets/src/styles/pressbooks-dashboard.scss

```diff
@@ -7,11 +7,20 @@
 .pb-dashboard-content {
   font-family: Karla, sans-serif;
 
+  h1 {
```

Comment:
These heading styles only apply to content inside .pb-dashboard-content containers. I will be using them for the forthcoming integrations self-activation page.